3_visualize/src/4_save.R

```diff
@@ -0,0 +1,23 @@
+# Save the processed data
+save_processed_data <- function(data){
+  out_dir="personal/training/pipelines/ds-pipelines-targets-1/3_visualize/out/"
+  readr::write_csv(data, file = file.path(out_dir, 'model_summary_results.csv'))
```

Comment:
We want to generally keep functions to producing one file at a time. I think this `write_csv()` line could be split into its own function, or added to your `make_eval_data()` function. You would need to update downstream functions to accept a filepath (or include the `read_csv()` step in your example of running the code).

Reply:
After I commit the changes I've made, they can both take in the data, so I don't think I need an extra read_csv step. But you tell me....